### ISSUE TITLE
Add accessories/metadata properties to game data

### ISSUE THREAD

**Issue by rsubtil** [retrohub-org/retrohub#302]
(suggested on Discord)

The idea is to add a property on game data in order to specify some details about that particular game, such as "lightguns, mouse, keyboard, rumble, 3 or 6 button, analog etc.".

This presents a challenge though: leaving it a plain "string" array for users to specify makes this non standardized for themes to take advantage of, but defining these properties in the app's code is also a significant task requiring plenty of research.

This idea has potential though as it can be help in organizing the user's library more easily, but the implementation of it needs to be better discussed and experimented on before being tackled.

**Comment by Ro-Borges** [comment]
Hello, it was me who suggested it. I don't know if it is possible, but maybe it could have a set of predefined/generic accessories that could be expanded with user-defined ones. This way, the themes would have a standard to comply with, and the users could customize their build with a more specific set if they desire.

**Comment by rsubtil** [comment]
Yes, I think that's a good solution for now. It will require some significant research and user input though, so this will be in the backlog for a while until the app is released and this receives more user feedback.
